### PR TITLE
Adding missing = in adapter client query creation

### PR DESCRIPTION
**Pull request by rohanahata**

### CODE REVIEW

**Comment by rohanahata** [comment]
https://issues.cask.co/browse/CDAP-2932

**Comment by anwar6953** [comment]
I know this is a easy bug fix, but can you also update AdapterClientTestRun.java, so that it would fail without this fix (and then pass with this change)?

File: cdap-client-tests/src/test/java/co/cask/cdap/client/AdapterClientTestRun.java [hunk 1]
```diff
@@ -86,6 +86,12 @@ public void testAdapters() throws Exception {
     adapterClient.waitForStatus(adapterName, AdapterStatus.STOPPED, 30, TimeUnit.SECONDS);
     adapterClient.start(adapterName);
     adapterClient.waitForStatus(adapterName, AdapterStatus.STARTED, 30, TimeUnit.SECONDS);
+
+    // The following test makes sure that if the request is malformed by the client, it will result in an error
```
Comment (anwar6953) [inline cdap-client-tests/src/test/java/co/cask/cdap/client/AdapterClientTestRun.java]: I don't think you need to specify this comment (describes a no longer existing bug). Just testing it is fine.

**Comment by anwar6953** [comment]
LGTM.
squash commits + merge, once build passes

**Comment by rohanahata** [comment]
http://builds.cask.co/browse/CDAP-DUT2182
https://issues.cask.co/browse/CDAP-2932

**Pull request merged**